.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.go]
+indent_style = tab
+
+[*.{js,jsx,json,html}]
+indent_style = space
+indent_size = 4
+
+[webapp/package.json]
+indent_size = 2
+
+[Makefile]
+indent_style = tab
```

Comment:
new line here :)

Reply:
Too bad it doesn't apply to its own configuration before it exists 😛